packages/d/distrobox/package.yml

```diff
@@ -0,0 +1,16 @@
+name       : distrobox
+version    : 1.7.2.1
+release    : 1
+source     :
+    - https://github.com/89luca89/distrobox/archive/refs/tags/1.7.2.1.tar.gz : ff2cca0c6334fff6ed577d23f68a6746ad4009f42d8a45eef5ca3850c895a4bb
+homepage   : https://distrobox.it/
+license    : GPL-3.0
+component  : virt
+summary    : Use any Linux distribution inside your terminal
+description: |
+    Use any Linux distribution inside your terminal. Enable both backward and forward compatibility with software and freedom to use whatever distribution you're more comfortable with
+rundeps   : podman
+install    : |
+    ./install -P $installdir
+    rm -r $installdir/share/zsh # for now. might create a subpackage for zsh autocomplete
```

Comment:
Why not leave it? Is it not working?

Reply:
I believe it works, but I don't use zsh. I didn't want more files to be installed if I am not going to use them.

Reply:
Other users might want to use it, please install them